### ISSUE TITLE
Can I try porting Sherlock> Termux?

### ISSUE THREAD

**Issue by torerobo** [sherlock-project/sherlock#387]
Android 6. Terminal Termux compilation error.

[LOG](https://pastebin.com/LLJpXJAM)

**Comment by sdushantha** [comment]
It works in Termux for me.
I saw that you got errors when you did:
```bash
python3 -m pip install -r requirements.txt
```
I also got that, so I had to install some of the requirements manually.
I think this problem has to do with Termux and `pip`, not Sherlock.
Let the dev of Termux know by creating an issue on one of their repos

----
Here is proof that Sherlock works on Termux :)
![Screenshot_20191201-103334](https://user-images.githubusercontent.com/27065646/69912211-42305d00-1426-11ea-9f22-64a7113eda9b.png)
